tema3/src/main/java/evidence/BubbleSort.java

```diff
@@ -0,0 +1,17 @@
+package evidence;
+
+public class BubbleSort {
+    public void sortedReps(Representative [] reps){
```

Comment:
This is not a bubble sort implementation. It is a brute force sorting. Please check the bubble sort algorithm.